sys-apps/ipmicfg/ipmicfg-1.27.1.170901.ebuild

```diff
@@ -0,0 +1,59 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit eapi7-ver
+
+MY_DATE="$(ver_cut 4)"
+MY_PN="${PN^^}"
+MY_PV="$(ver_cut 1-3)"
+
+DESCRIPTION="IPMICFG is an in-band utility for configuring IPMI devices"
+HOMEPAGE="https://www.supermicro.com"
+SRC_URI="ftp://ftp.supermicro.com/utility/${MY_PN}/${MY_PN}_${MY_PV}_build.${MY_DATE}.zip"
```

Comment:
add `app-arch/unzip` to `DEPEND` as `SRC_URI` pointed to the `.zip` archive

Reply:
Added by newest push.